openeo/rest/connection.py

```diff
@@ -1385,6 +1385,56 @@ def load_stac(
             _log.warning(f"Failed to extract cube metadata from STAC URL {url}", exc_info=True)
         return cube
 
+    def load_stac_from_job(
+        self,
+        job: Union[BatchJob, str],
+        spatial_extent: Optional[Dict[str, float]] = None,
+        temporal_extent: Union[Sequence[InputDate], Parameter, str, None] = None,
+        bands: Optional[List[str]] = None,
+        properties: Optional[Dict[str, Union[str, PGNode, Callable]]] = None,
+    ) -> DataCube:
+        """
+        Wrapper for :py:meth:`load_stac` that loads the result of a previous job using the STAC collection of its results.
+
+        :param job: The batch job object that has successfully finished, or a job id. When a job id is provided,
+            a job object is created using self as connection.
+        :param spatial_extent: limit data to specified bounding box or polygons
+        :param temporal_extent: limit data to specified temporal interval.
+        :param bands: only add the specified bands
+        :return: a :py:class:`DataCube`
```

Comment:
Only add a `:return:` doc when you document more than just the return type. 
The return type is already covered by the type annotation, and rendered in the docs, so a `:return:` line like this gives duplication there